test/integration/edition_edit_test.rb

```diff
@@ -56,6 +58,30 @@ class EditionEditTest < IntegrationTest
         assert_selector(".govuk-summary-list__value", text: @draft_edition.assignee)
       end
     end
+
+    should "display the important note if an important note exists" do
+      note_text = "This is really really urgent!"
+      create_draft_edition
+      create_important_note_for_edition(@draft_edition, note_text)
+      visit edition_path(@draft_edition)
+
+      assert page.has_text?("Important")
+      assert page.has_text?(note_text)
```

Comment:
Is it worth adding a test that these two text elements are within a banner?
So maybe 
```
within :css, ".govuk-notification-banner" do
  assert page.has_text?("Important")
  assert page.has_text?(note_text)
end
```